app/src/main/java/edu/cascadia/mobile/apps/layoutexamples/MainActivity.java

```diff
@@ -40,4 +40,11 @@ public void onCheckboxClicked(View view) {
         //Display the string as a temporary popup - called a "Toast"
         Toast.makeText(MainActivity.this, teaTime, Toast.LENGTH_SHORT).show();
     }
+
+    public void onClick(View view) {
+        String teaWith = "Tea with ";
+        teaWith = teaWith + TeaGoodies;
```

Comment:
consider reusing the teaTime stringBuilder, by making it an instance variable.

Reply:
Missing TODO assignment of onClickListeners on line 24
`sugarCheck.setOnClickListener( v -> onCheckboxClick(v) );`